draft-ietf-quic-http.md

```diff
@@ -1202,9 +1202,15 @@ The following situations are examples of invalid PRIORITY frames:
 A PRIORITY frame with Empty bits not set to zero MAY be treated as a connection
 error of type HTTP_MALFORMED_FRAME.
 
-A PRIORITY frame that references a non-existent Push ID, a Placeholder ID
-greater than the server's limit, or a Stream ID the client is not yet permitted
-to open MUST be treated as a connection error of type HTTP_LIMIT_EXCEEDED.
+A PRIORITY frame that references a non-existent Push ID, or a Stream ID the
+client is not yet permitted to open MUST be treated as a connection error of
+type HTTP_LIMIT_EXCEEDED.
+
+A PRIORITY frame MAY reference a Placeholder ID that is equal to or greater than
+the server's advertised value of the `SETTINGS_NUM_PLACEHOLDERS` settings.  The
+server SHOULD ignore PRIORITY frames that specify such placeholders as the
+prioritized element.  The server SHOULD use the root of the tree as the
```

Comment:
I think you want s/prioritized element/element dependency. Such that if a stream is assigned as dependent on a placeholder ID greater than SETTINGS_NUM_PLACEHOLDERS, then it becomes dependent on root.

Reply:
NVM, you say that in the next sentence. 

Perhaps if you reorder the text to make the subject of each sentence come earlier, it will be easier to read it correctly.

Reply:
@LPardue Thank you for the comment. I've changed the flow of the sentences in 4579ada. PTAL.

Reply:
That looks better now.